src/Common/src/Interop/Windows/advapi32/Interop.ReadEventLog.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Advapi32
+    {
+        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool ReadEventLog(
+            SafeHandle hEventLog,
+            int dwReadFlags,
+            int dwRecordOffset,
+            byte[] buffer,
+            int numberOfBytesToRead,
```

Comment:
Nit: `numberOfBytesRead` => `nNumberOfBytesRead` to match the doc'd parameter name